cmd/sqlflowserver/main_test.go

```diff
@@ -529,7 +529,7 @@ func CaseTrainSQL(t *testing.T) {
 	a := assert.New(t)
 	trainSQL := fmt.Sprintf(`SELECT *
 FROM %s.%s
-TRAIN DNNClassifier
+TO TRAIN DNNClassifier
```

Comment:
There are so many changes in this file. This reminds us to refactoeize this file somehow to remove duplications.